types/index.d.ts

```diff
@@ -836,6 +836,25 @@ declare module 'cloudinary' {
         value: string;
     }
 
+    export type AnalysisType = 'captioning' | 'cld_fashion' | 'cld_text' | 'coco' | 'google_tagging' | 'human_anatomy' | 'lvis' | 'shop_classifier' | 'unidet' | 'custom';
+
+    export interface AnalyzeResponse {
+        data: {
+            entity: string,
+            analysis: Record<string, Record<string, string> | Array<string> | string>
+        },
+        request_id: string,
+    }
+
+    export interface AnalyzeOptions {
+        analyze_parameters: {
```

Comment:
Shouldn't it be just `parameters`? or does it not matter?

Reply:
I changed the interface a little bit